Documentation/batch-scripts/mentoring.py

```
@@ -0,0 +1,436 @@
+)
+
+def push_event(event: dict):
+    """Push JSON event to Kafka"""
+    producer.send(TOPIC, event)
+    producer.flush()
+
+# ---------------- DB Setup ----------------
+SOURCE_DB1 = dict(
+    dbname="mentoring",
+    user="shikshalokam",
+    password="Admin@123",
```

Ruff (0.13.1)

Possible hardcoded password assigned to argument: "password"

(S106)